openhub/management/commands/import_affiliated_committers_data.py

```diff
@@ -0,0 +1,74 @@
+import json
+import requests
+import logging
+import xmltodict
+
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+from django.core.management.base import BaseCommand
+from openhub.models import MostCommit, MostRecentCommit, AffiliatedCommitter
+
+logging.basicConfig(level=logging.DEBUG)
+
+n = 100
+for i in range(1, 2):
```

Comment:
this is effectively invalid syntax.  You replace the module level `Command` on the second iteration

no, the logic, all of it, needs to be modules in `openhub/` .  The `Command` *must* be at the top level of this module, not in any `if` or loop, and must only invoke the logic in `openhub/` modules (not subdirectories under that).  See the GCI app `Command` for an example.